src/test/java/com/ft/methodearticlemapper/messaging/MessageProducingArticleMapperTest.java

```diff
@@ -39,15 +43,15 @@
 
     @Before
     public void setUp() {
-        msgProducingArticleMapper = new MessageProducingArticleMapper (
+        msgProducingArticleMapper = new MessageProducingArticleMapper(
                 messageBuilder,
                 producer,
                 mapper
         );
     }
 
     @Test
-    public void thatMessageIsCreatedFromMappedArticle() {
+    public void thatMessageIsCreatedFromMappedArticle() throws ParserConfigurationException, SAXException, XPathExpressionException, IOException {
```

Comment:
Don't forget to remove the throws clause from these tests as well after removing it from the `mapArticle` method.